java/runtime/src/main/java/org/ray/runtime/task/ArgumentsBuilder.java

```diff
@@ -6,14 +6,13 @@
 import org.ray.api.RayActor;
 import org.ray.api.RayObject;
 import org.ray.api.id.UniqueId;
+import org.ray.runtime.AbstractRayRuntime;
 import org.ray.runtime.util.Serializer;
 
 public class ArgumentsBuilder {
 
-  private static boolean checkSimpleValue(Object o) {
-    // TODO(raulchen): implement this.
-    return true;
-  }
+  private static final int LARGEST_SIZE_PASS_BY_VALUE = 100 * 1024;
```

Comment:
constants should really go in https://github.com/ray-project/ray/blob/master/src/ray/constants.h or https://github.com/ray-project/ray/blob/master/src/ray/ray_config.h.

Reply:
What's involved in doing that? It is analogous to these parameters

https://github.com/robertnishihara/ray/blob/57d6e983022ad86559b07f6a863404198ebf2347/src/ray/ray_config.h#L218-L221

One advantage of putting parameters in the config file is that they can be changed at runtime without recompiling.

Reply:
To read constants from C++, we'll need to use cumbersome JNI (e.g., something like [this](https://github.com/ray-project/ray/blob/master/src/ray/raylet/lib/java/org_ray_runtime_raylet_RayletClientImpl.cc)).

If you prefer changing this number dynamically, we can put this number in [RayConfig.java](https://github.com/ray-project/ray/blob/master/java/runtime/src/main/java/org/ray/runtime/config/RayConfig.java).

BTW, how can we dynamically change ray_config.h? For raylet, we can use the `internal_config` parameter. But for Python worker, is there a similar API?

Reply:
@raulchen good point, I think it is only possible to change it for the raylet currently.